src/mpyl/steps/deploy/bpmn.py

```diff
@@ -0,0 +1,97 @@
+"""Deploys the Camunda diagrams in the build stage to Camunda cluster, using BPM. """
+import os
+from logging import Logger
+from python_on_whales import docker, Container, DockerException
+from ...utilities.docker import execute_with_stream
+from ...utilities.bpmn import CamundaConfig
+from . import STAGE_NAME
+from .. import Step, Meta
+from ..models import Input, Output, ArtifactType
+
+
+class BpmnDiagramDeploy(Step):
+    def __init__(self, logger: Logger) -> None:
+        super().__init__(
+            logger,
+            Meta(
+                name="BPMN Diagram Deploy",
+                description="Deploy BPMN diagram to Camunda Cluster",
+                version="0.0.1",
+                stage=STAGE_NAME,
+            ),
+            produced_artifact=ArtifactType.NONE,
+            required_artifact=ArtifactType.NONE,
+            # TO DO: make an artifcatype with a camunda deployed diagram link
+        )
+
+    def execute(self, step_input: Input) -> Output:
+        volume_path = os.path.join(os.getcwd(), "bpm")
+        camunda_config = CamundaConfig.from_config(
+            step_input.run_properties.config, step_input.run_properties.target
+        )
+        docker_container = self._get_docker_container(volume_path, camunda_config)
+        bpm_file_path = volume_path + "/camunda-diagrams/src/test/resources/"
```

Comment:
this as well? you don't want to have to release a new mpyl version when a folder path changes in the monorepo